File: Backend/API/Shared/Dtos/VehicleInsuranceDetailsUpdateModel.cs
using Shared.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Shared.Dtos
{
    public class VehicleInsuranceDetailsUpdateModel
    {
        public int VehicleId { get; set; }
        public Vehicle? Vehicle { get; set; }
        public List<VehicleInsuranceDetails> VehicleInsuranceDetails { get; set; }

    }
}

File: Frontend/Admin/src/app/shared/models/vehicleInsuranceDetailsUpdateModel.ts
import { Discount } from "./discount";
import { Vehicle } from "./vehicle";
import { VehicleInsuranceDetail } from "./vehicleInsuranceDetail";

export class VehicleInsuranceDetailsUpdateModel {
  vehicleId: number;
  vehicle: Vehicle;
  vehicleInsuranceDetails: VehicleInsuranceDetail[];
}
